apps/chatbot/src/env.ts

```diff
@@ -1,7 +1,7 @@
-import { LogLevel } from "@twurple/chat";
 import { createEnv } from "@t3-oss/env-core";
-import { z } from "zod";
+import { LogLevel } from "@twurple/chat";
 import "dotenv/config";
+import { z } from "zod";
```

Comment:
Is there a setting so that imports that aren't importing anything (those that just affect the global scope) are sorted last?

Reply:
I don't think so, but I'll do some research

Reply:
Nope, no setting that supports that